server/routes/v1/deployment.ts

```diff
@@ -111,3 +113,28 @@ export const postDeployment = [
     })
   },
 ]
+
+export const resetDeploymentApprovals = [
+  ensureUserRole('user'),
+  bodyParser.json(),
+  async (req: Request, res: Response) => {
+    const user = req.user
+    const body = req.body as any
+    const deployment: Deployment = await DeploymentModel.findOne({ uuid: body.uuid })
+    if (user?.id !== deployment.metadata.contacts.requester) {
```

Comment:
I'm worried about id changes, whenever that becomes a requirement.  Please could we compare the database id with the deployment owner (e.g. `user?._id !== deployment.owner`).